gallery/src/demos/demo-ha-chip-set.ts

```diff
@@ -0,0 +1,34 @@
+import { css, html, LitElement, TemplateResult } from "lit";
+import { customElement } from "lit/decorators";
+import "../../../src/components/ha-card";
+import "../../../src/components/ha-chip-set";
+
+const chips: string[] = ["Chip 1", "Chip 2", "Chip 3", "Chip 4"];
+
+@customElement("demo-ha-chip-set")
```

Comment:
I think it would be fine to have all chip stuff on 1 page, just do the chipset in a new card.